monai/networks/nets/hovernet.py

```diff
@@ -423,10 +398,17 @@ def __init__(
 
         super().__init__()
 
-        self.mode: int = self._mode_to_int(mode)
+        if isinstance(mode, HoVerNetMode):
+            self.mode = mode.value
```

Comment:
this could be improved by using https://github.com/Project-MONAI/MONAI/blob/ba670c9ba78f9f4d76eb12b4d33f245b5774d462/monai/utils/module.py#L58-L61